setup.py

```diff
@@ -61,5 +61,8 @@ def parse_requirements():
         "mamba-ssm": [
             "mamba-ssm==1.0.1",
         ],
+        "autogptq": [
+            "auto-gptq==0.5.1 --extra-index-url https://huggingface.github.io/autogptq-index/whl/cu118/",
```

Comment:
keep `--extra-index-url https://huggingface.github.io/autogptq-index/whl/cu118/` in the requirements.txt, and this line is simply 
```suggestion
            "auto-gptq==0.5.1",
```